snet_cli/commands.py

```diff
@@ -263,22 +263,40 @@ def set(self):
         self.config.set_session_network(self.args.network_name, self.out_f)
 
 
-class SessionCommand(Command):
+class SessionSetCommand(Command):
     def set(self):
         self.config.set_session_field(self.args.key, self.args.value, self.out_f)
 
     def unset(self):
         self.config.unset_session_field(self.args.key, self.out_f)
 
+class SessionShowCommand(BlockchainCommand):
     def show(self):
         rez = self.config.session_to_dict()
+        key =  "network.%s"%rez['session']['network']
+        self.populate_contract_address(rez, key)
 
         # we don't want to who private_key and mnemonic
         for d in rez.values():
             d.pop("private_key", None)
             d.pop("mnemonic", None)
         self._pprint(rez)
 
+    def populate_contract_address(self, rez, key):
+        try:
+            rez[key]['default_registry_address'] = read_default_contract_address(w3=self.w3, contract_name="Registry",
+                                                     err_msg="unable to fetch registry address")
+            rez[key]['default_multiparty_escrow_address'] = read_default_contract_address(w3=self.w3, contract_name="MultiPartyEscrow",
```

Comment:
For sake of consistency I would use the following names: ```default_multipartyescrow_at``, ```default_registry_at```,  ```default_singularitynettoken_at``` (because we already have ```current_multipartyescrow_at```, ```current_registry_at```, ```current_singularitynettoken_at```).